community/primitive-collections/src/test/java/org/neo4j/collection/primitive/hopscotch/BasicTableTest.java

```diff
@@ -149,6 +149,26 @@ public Object sampleValue()
             }
         } } );
         result.add( new Object[] { new TableFactory()
+        {
+            @Override
+            public Table newTable( int capacity )
+            {
+                return new LongKeyLongValueTable( capacity );
+            }
+
+            @Override
+            public boolean supportsLongs()
+            {
+                return true;
+            }
+
+            @Override
+            public Object sampleValue()
+            {
+                return new long[] {Math.abs( random.nextLong() )};
```

Comment:
Why the `Math.abs`? The `LongKeyIntValueTable` above doesn't do this. I guess you were following the steps of the off-heap table?

Reply:
I also though that was strange, but actually the `LongKeyIntValueTable` also does this, because of the `random.nextInt( Integer.MAX_VALUE )`. Not allowing negative integers is a bit surprising. 

Don't know if it's related to the magical unicorn `LongKeyIntValueTable#NULL = -1`?

Reply:
That's probably it.